### PR TITLE
Update to latest Oldowan; convert build to TypeScript

### PR DESCRIPTION
**Pull request by ashmind**

### CODE REVIEW

File: WebAssets/package.json [hunk 1]
```diff
@@ -59,13 +59,13 @@
...
         "eslint": "^6.8.0",
-        "execa": "^4.0.0",
         "fast-glob": "^3.2.2",
         "fs-jetpack": "^2.1.0",
         "jest": "^25.1.0",
         "keysim": "^2.0.0",
-        "oldowan": "^1.0.0-preview-2",
+        "oldowan": "^1.0.0-preview-3",
         "ts-jest": "^25.2.1",
+        "ts-node": "^8.9.0",
```
Comment (jsk95) [inline WebAssets/package.json:68]: where do you keep finding libraries like this

Comment (jsk95) [reply WebAssets/package.json:68]: Not sure if I like it

Comment (ashmind) [reply WebAssets/package.json:68]: I don't see an alternative as deno can't be installed through npm?

Comment (jsk95) [reply WebAssets/package.json:68]: actually would be nice if ms shipped this themselves

**Pull request merged**